template/.pre-commit-config.yaml.j2

```diff
@@ -42,3 +42,12 @@ repos:
         entry: make regenerate-charts
         stages: [commit, merge-commit, manual]
         pass_filenames: false
+
+  - repo: local
+    hooks:
```

Comment:
Lets combine it with the above repo. `hooks` accepts a list of hooks and thus `cargo-test` can be combined with `regenerate-charts` above.

Reply:
Good idea. Done.